martian/src/stage.rs

```diff
@@ -164,6 +166,11 @@ impl Resource {
         self.threads
     }
 
+    /// Get the special resource request
+    pub fn get_special(&self) -> Option<String> {
+        self.special.clone()
```

Comment:
Could leave the clone to the caller
```suggestion
    pub fn get_special(&self) -> Option<&str> {
        self.special.as_ref()
```